projects/laji/src/app/+observation/observation.facade.ts

```diff
@@ -100,7 +95,7 @@ export class ObservationFacade {
     private warehouseApi: WarehouseApi,
     private observationDataService: ObservationDataService
   ) {
-    this.updateState({..._state, ...this.persistentState});
+    this.updateState({..._state});
```

Comment:
This is a bit out of the scope if this PR, but do you know why we have the singleton `_state` variable? The variable outlives the component lifecycle, so if you e.g. navigate away from the obs page and navigate back, the component state is initialized here with the `_state` var which could still hold the old state. I don't know why we haven't ran into bugs with this, are we maybe clearing the `_state` when the component page is destroyed? Regardless of how it manages to work, I don't see any reason why it's a singleton. If there's no reason for having it, I anticipate the logic would be simpler without it.

Reply:
Hmm, can't think any reason for it. Might be an old relic. I guess it doesn't cause any problems since it sets the correct state when it loads the query from url parameters